### PR TITLE
Ruotong's Regex Exercise Post

### PR DESCRIPTION
**Pull request by izayak**

### CODE REVIEW

File: _posts/izayak/2016-03-29-regex-exercise-ry.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+author: izayak
+title: "Ruotong's Regex Exercise
```
Comment (eah13) [inline _posts/izayak/2016-03-29-regex-exercise-ry.md]: You'll need a `"` to end this title

File: _posts/izayak/2016-03-29-regex-exercise-ry.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+<iframe src="https://trinket.io/embed/python/b9c50ef805" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+  
+**Reflection**
+The first two exercises help us practice the methods discussed in the textbook which can be quite useful. The re.search() and re.findall() are two tools that are helpful when people search through files for certain contents. The third exercise help us review the materials of exercise 4 of last assignment and how to convert a list to a string. These exercises deepen our understanding of strings and lists and require a comprehension of these two main sturctures. 
+
+  
+*Question*
+I first tried to do this in Python 3 of Trinket, but it cannot be shared, why?
```
Comment (eah13) [inline _posts/izayak/2016-03-29-regex-exercise-ry.md:15]: You'll need to make one from trinket.io/python3 for the share button to be active.  Please replace this trinket with one made from there.

**Comment by izayak** [comment]
@eah13 Hope it works now! Thanks!

**Comment by eah13** [comment]
I'll go ahead and merge but this still isn't working.  It needs to be a Python 3 trinket with the files attached.  The most recent link you included isn't a Python 3 trinket and throws an error when I try to run it.

**Pull request merged**